ampd-handlers/src/bin/evm/main.rs

```diff
@@ -116,42 +116,23 @@ fn build_event_verifier_handler(
     Ok(handler)
 }
 
-fn init_tracing(max_level: Level) {
-    let error_layer = ErrorLayer::default();
-    let filter_layer = EnvFilter::builder()
-        .with_default_directive(LevelFilter::from_level(max_level).into())
-        .from_env_lossy();
-    let fmt_layer = tracing_subscriber::fmt::layer().json().flatten_event(true);
-
-    tracing_subscriber::registry()
-        .with(error_layer)
-        .with(filter_layer)
-        .with(fmt_layer)
-        .init();
-}
-
 #[tokio::main]
 async fn main() -> Result<(), Error> {
-    init_tracing(Level::INFO);
+    ampd_handlers::tracing::init_tracing(Level::INFO);
 
     let base_config = config::Config::from_default_sources().change_context(Error::HandlerStart)?;
 
-    let gmp_handler_config = config::Config::builder()
+    // Load combined EVM handler config
+    let handler_config = config::Config::builder()
         .add_file_source("evm-handler-config.toml")
         .add_env_source("AMPD_EVM_HANDLER")
-        .build::<EvmGmpHandlerConfig>()
-        .ok();
-
-    let event_verifier_handler_config = config::Config::builder()
-        .add_file_source("evm-event-verifier-handler-config.toml")
-        .add_env_source("AMPD_EVM_EVENT_VERIFIER_HANDLER")
-        .build::<EvmEventVerifierHandlerConfig>()
-        .ok();
+        .build::<EvmHandlerConfig>()
+        .change_context(Error::HandlerStart)?;
 
     // Error if no handlers are configured
-    if gmp_handler_config.is_none() && event_verifier_handler_config.is_none() {
+    if handler_config.handlers_to_run.is_empty() {
         return Err(Error::HandlerStart)
-            .attach_printable("No handler configurations found. At least one of evm-handler-config.toml or evm-event-verifier-handler-config.toml must be present");
+            .attach_printable("No handlers configured. The 'handlers_to_run' array in evm-handler-config.toml must contain at least one handler (gmp_voting or event_verification)");
```

Comment:
have in mind that `evm-handler-config.toml` is optional and the whole config could be provided using env variables